system-tests/projects/attachFile/yarn.lock

```diff
@@ -0,0 +1,163 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
this is missing the accompanying package.json

so this means that https://github.com/cypress-io/cypress/pull/18574 blocks this PR, right?

Reply:
Mistake on my part actually - I've added the necessary dependencies to `system-tests/package.json`, and this just accidentally included from an earlier way of doing things. Removing.